auth/README.md

```diff
@@ -55,24 +55,25 @@ The service is configured using the environment variables presented in the
-| MF_AUTH_GRPC_PORT         | Auth service gRPC port                                                   | 8181          |
-| MF_AUTH_SERVER_CERT       | Path to server certificate in pem format                                 |               |
-| MF_AUTH_SERVER_KEY        | Path to server key in pem format                                         |               |
-| MF_AUTH_SECRET            | String used for signing tokens                                           | auth          |
-| MF_JAEGER_URL             | Jaeger server URL                                                        | localhost:6831|
+| Variable                      | Description                                                              | Default        |
+|-------------------------------|--------------------------------------------------------------------------|----------------|
+| MF_AUTH_LOG_LEVEL             | Service level (debug, info, warn, error)                                 | error          |
+| MF_AUTH_DB_HOST               | Database host address                                                    | localhost      |
+| MF_AUTH_DB_PORT               | Database host port                                                       | 5432           |
+| MF_AUTH_DB_USER               | Database user                                                            | mainflux       |
+| MF_AUTH_DB_PASSWORD           | Database password                                                        | mainflux       |
+| MF_AUTH_DB                    | Name of the database used by the service                                 | auth           |
+| MF_AUTH_DB_SSL_MODE           | Database connection SSL mode (disable, require, verify-ca, verify-full)  | disable        |
+| MF_AUTH_DB_SSL_CERT           | Path to the PEM encoded certificate file                                 |                |
+| MF_AUTH_DB_SSL_KEY            | Path to the PEM encoded key file                                         |                |
+| MF_AUTH_DB_SSL_ROOT_CERT      | Path to the PEM encoded root certificate file                            |                |
+| MF_AUTH_HTTP_PORT             | Auth service HTTP port                                                   | 8180           |
+| MF_AUTH_GRPC_PORT             | Auth service gRPC port                                                   | 8181           |
+| MF_AUTH_SERVER_CERT           | Path to server certificate in pem format                                 |                |
+| MF_AUTH_SERVER_KEY            | Path to server key in pem format                                         |                |
+| MF_AUTH_SECRET                | String used for signing tokens                                           | auth           |
+| MF_AUTH_LOGIN_TOKEN_DURATION  | Time in minutes for the login token to last of type time.duration        | 600m           |
```

Comment:
Please use `10h` instead of `600m` here.

Reply:
Done